### DISCUSSION TITLE
Budi cli Port 6379 Connection Errors (but docker-compose works fine)

### DISCUSSION THREAD

**Discussion by pbarmak** [Budibase/budibase#13269] [Help] [answered]
I'm trying to set up a new Azure Ubuntu 22.04 instance. Following the Budibase CLI docs, I installed docker, docker-compose, and node (version 18, since v14 that the docs say to use is very old per the node installer).

When I then go to run` budi hosting --init`, I get a ton of the following errors and I cannot even ctl-c anything (have to kill from another terminal):
```
[11:39:04.738] ERROR (5840): [BULL] doc-writethrough=error {"tenantId":"default","bull":{"eventType":"doc-writethrough","event":"error"}}
    err: {
      "type": "Error",
      "message": "connect ECONNREFUSED 127.0.0.1:6379",
      "stack":
          Error: connect ECONNREFUSED 127.0.0.1:6379
              at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
      "errno": -111,
      "code": "ECONNREFUSED",
      "syscall": "connect",
      "address": "127.0.0.1",
      "port": 6379
    }
```

I'm not sure what I'm missing. I have inbound ports 10000,4001,9000,8080,4369,5984,9100,6379 open. I have budibase running on an Ubuntu 20.04 instance (node version 16). I'm sure it's something simple that I'm just not catching.

The funny thing is, if I set up for docker-compose (yaml file, .env file), I can get `docker up` to work fine - it loads all services. But I still cannot get the cli to run and I'd love to have that as I use it for exports/imports.

I have tried the install on arm64 and x64. Both have the same error. I prefer to use arm64 if possible.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @pbarmak,

Thanks for creating this discussion we are aware of this issue and are looking into fixing it. There seems to be an issue currently with the CLI. Thank you for also creating an issue. Will [link](https://github.com/Budibase/budibase/issues/13276) it here incase anyone else stumbles on this discussion.
